client/connection/openvpn.go

```diff
@@ -57,3 +64,18 @@ func ConfigureVpnClientFactory(
 		), nil
 	}
 }
+
+func detectCountry(ipResolver ip.Resolver, locationDetector location.Detector) (string, error) {
```

Comment:
This function can live on it's own package, it's not related to openvpn anyhow

Reply:
Plus its duplicated currently

Reply:
now this function lives in location package